esx_service/utils/auth.py

```diff
@@ -91,34 +91,34 @@ def get_default_tenant():
         return None, None, None
 
 
-def get_default_privileges():
-    """ Get DEFAULT privileges by querying the auth DB.
-        DEFAULT privilege will match any (tenant_uuid, datastore) pair which
-        does not have an entry in privileges table explicitly
+def get_all_ds_privileges(tenant_uuid):
+    """ Get privileges on _ALL_DS by querying the auth DB.
+        For the tenant with given tenant_uuid, this privilege
+        will match any datastore which does not have an entry
+        in privileges table explicitly
```

Comment:
Ok, my doubt was if a user in VMG1 will be allowed to create on DS2 just with the __ALL_DS privilege, without explicitly adding DS2 to VMG1. . Which seems to be allowed. 

Is it possible to update the documentation for the predefined datastore names (VM_DS and ALL_DS) and the privileges allowed for these,. And the default or typical use cases when these must be used.